xml/System.Buffers.Text/Utf8Formatter.xml

```diff
@@ -51,13 +51,27 @@
         <Parameter Name="format" Type="System.Buffers.StandardFormat" />
       </Parameters>
       <Docs>
-        <param name="value">To be added.</param>
-        <param name="destination">To be added.</param>
-        <param name="bytesWritten">To be added.</param>
-        <param name="format">To be added.</param>
-        <summary>To be added.</summary>
-        <returns>To be added.</returns>
-        <remarks>To be added.</remarks>
+        <param name="value">The value to format.</param>
+        <param name="destination">The buffer to write the UTF8-formatted value to.</param>
+        <param name="bytesWritten">When the method returns, contains the length of the formatted text in bytes.</param>
+        <param name="format">The standard format to use.</param>
+        <summary>Formats a <see cref="T:System.Boolean" /> as a UTF8 string.</summary>
```

Comment:
UTF8 should generally be written as "UTF-8", with the hyphen.

https://en.wikipedia.org/wiki/UTF-8#Official_name_and_variants

This applies everywhere (and looks like we follow that in other, existing API docs):
https://docs.microsoft.com/en-us/dotnet/api/system.text.encoding.utf8?view=netframework-4.8

Reply:
Same concern here with using "UTF-8 string" for `ROS<byte>`

cc @bartonjs, @GrabYourPitchforks